### ISSUE TITLE
Creating a SVM for achieving higher accouracy (98%)

### ISSUE THREAD

**Issue by theshredbox** [srimani-programmer/Breast-Cancer-Predictor#165]
Several methods show that ML techniques can show high accuracy levels ranging rom 90%- 97%. Since we have used RFR in the main branch, I'll be creating a SVM to show much higher accuracy.
Also, I'll be improving the model by normalizing the training data which is a scaling feature process. I'll be further searching hyperparameters for improving the model. 

I want to work on this issue under GSSoC 22.

**Comment by freny24** [comment]
@theshredbox Issue has been assigned to you ! Kindly work on it.

**Comment by nicolemd7** [comment]
Hi I have done this in an SVC model pull request #185 in which I have achieved  98% accuracy
